### ISSUE TITLE
String arrays not initialized when Colors.wurst is used in @compiletime

### ISSUE THREAD

**Issue by IgorSamurovic** [wurstscript/WurstStdlib2#9]
Basically, I am doing automatic generation of tooltips and I am converting numbers to hex in order to produce color codes. It works out of compiletime, just fine.

However, in compiletime, I get the following error:

> You encountered a bug in the interpreter: java.lang.ClassCastException: de.peeeq.wurstscript.intermediatelang.ILconstNull cannot be cast to de.peeeq.wurstscript.intermediatelang.ILconstAddable

After a bit of testing I realized that this happens because hexs[ ] array isn't initialized at all. Its values are normally initialized in Colors.wurst init block, but that block is never ran in compiletime, meaning that it is impossible to use toHex function from compiletime.

If I create a @compiletime function that initializes those values, and still keep it in Colors.wurst, it does not solve the problem at all, it seems that wurst won't call @compiletime functions from packages that are not explicitly imported, and I can't do that with Colors.wurst because it is already included in every namespace, so my explicit import is ignored.

The "hack" solution I've found for this problem just to have my code work is to make a public function in Colors.wurst that initializes hexs[ ] array, and then call it from my own compiletime function just before I know toHex function is going to be used, but this is obviously not any sort of a sensible solution.

I am aware that Colors.wurst probably isn't supposed to be used in compiletime, but this might be a more general issue.

TL;DR: Packages that are imported everywhere by default don't seem to call the init block in @compiletime which causes some things that depend on package-level variables to fail.

**Comment by Frotty** [comment]
Hey Tauren, thanks for the report. Though this is more of a general compiletime problem rather than stdlib.
Will investigate.

**Comment by Frotty** [comment]
Refer to https://github.com/wurstscript/WurstScript/issues/574
